src/components/state-history-charts.html

```diff
@@ -14,6 +14,14 @@
       display: block;
     }
 
+    .google-visualization-tooltip {
+      z-index: 200;
+    }
+
+    state-history-chart-timeline, state-history-chart-line {
```

Comment:
I wonder if we should have this be contained in the respective web component definitions?

Reply:
Both of those clear inner DOM and apply the graph object to `this`, so any style would get deleted.

It is possible to apply the graph object to some inner div, but I think of state-history-chart-* as implementation details and not really standalone web components.